### DISCUSSION TITLE
Negative time-lag

### DISCUSSION THREAD

**Discussion by gnarwhal97** [NVlabs/sionna#670] [Q&A]
Why in time_lag_discrete_time_channel The smallest time-lag returned is always -6? Could you explain what is time lag and why is it minimum value is negative? Thank you!

**Comment by jhoydis** [comment]
Hi @gnarwhal97,

When you low-pass filter the channel impulse response with a sinc-filter, you get an infinitely long result (in both positive and negative time directions) which must be truncated.
We have observed empirically that you rarely see any energy for taps beyond -6. That's why this value was chosen.
